dashboard/sample_metadata_classifier.py

```diff
@@ -231,5 +231,11 @@ def interpret(project, papers, bits):
             #  admission.
             date="2020-01",
             collection="bronchoalveolar lavage fluid")
+    elif project in papers["Riquelme 2022"]["projects"]:
+        sample,wtp,country,date = bits
```

Comment:
This is less obviously true now than it was a few years ago, but I think Hong Kong and mainland China are different enough to be worth treating separately? This seems to be moderately common practise, though I expect it will lapse eventually.